examples/02-plot/themes.py

```diff
@@ -166,10 +166,28 @@ def plot_example():
 # Alternatively, set the theme of an instance of ``Plotter``.
 
 pl = pv.Plotter(theme=my_theme)
-# pl.theme = my_theme  # alternatively use the setter
 pl.add_mesh(pv.Cube())
 cpos = pl.show()
 
+# %%
```

Comment:
The added line below are referencing a now deprecated functionality.  I would argue we should not add an example for a deprecated feature.